src/common/utils.jsx

```diff
@@ -19,36 +20,11 @@ FYABackend.interceptors.response.use(
   },
 );
 
-// Reference auth_utils.js for an additional auth interceptor.
-
-export const isValidZip = zip => {
-  const countries = [
-    'US',
-    'AT',
-    'BG',
-    'BR',
-    'CA',
-    'CZ',
-    'DK',
-    'FR',
-    'DE',
-    'IN',
-    'IT',
-    'IE',
-    'MA',
-    'NL',
-    'PL',
-    'PT',
-    'RO',
-    'RU',
-    'SG',
-    'SK',
-    'ES',
-    'SE',
-    'CH',
-    'GB',
-  ];
-  return countries.filter(country => isValidZipcode(zip, country)).length > 0;
+export const isValidZip = zipCode => {
+  const countries = countryList().getValues();
+  return (
+    countries.filter(countryCode => postalCodes.validate(countryCode, zipCode) === true).length > 0
```

Comment:
Have you tested any mismatching zip codes and countries (i.e. zip codes that do not belong in the country) to ensure that this postal codes library is working as intended?

Reply:
yep! for example, if you put 324 for zipcode and select United States for country, it'll shown an error message saying "Postal code 324 is not valid for country US"